src/sass/_Fabric.i18n.scss

```diff
@@ -0,0 +1,315 @@
+// Copyright (c) Microsoft. All rights reserved. Licensed under the MIT license. See LICENSE in the project root for license information.
+
+//
+// Office UI Fabric
+// --------------------------------------------------
+// Internationalization mixins
+
+
+// Base left-to-right mixin definition.
+@mixin LTR {
+  html[dir='ltr'] & {
```

Comment:
The overrides can become a little tricky in some specific inheritance scenarios, but I agree. Following up with some eng partners who helped make this in the first place.